config/site_settings.yml

```diff
@@ -2059,9 +2059,13 @@ tags:
     default: false
     client: true
   remove_muted_tags_from_latest:
-    default: false
-  mute_other_present_tags:
-    default: true
+    client: true
+    type: enum
+    default: always
+    choices:
+      - never
+      - when used alone or with other muted tags
```

Comment:
This name is cumbersome. I think we should define an enum with translations, so that the id itself can be one word (or two word with snake case). It keeps the code simpler and translators can change it.